### ISSUE TITLE
[In-app dapp webview] Option to open in external browser

### ISSUE THREAD

**Issue by kathaypacific** [valora-inc/wallet#2360]
### Description
Provide users the option to open dapps in an external browser, for several reasons
- if for whatever reason the in-app webview is not working correctly then we should provide an escape hatch (e.g. we saw the connection issue with Android in the previous release, where users were unable to connect to any dapps)
- allows us to measure the success of the in-app browser, by seeing how many users prefer to exit out of it

https://www.figma.com/file/zdK8XvxNXvTjxnM5V33pHw/Dapps-Page?node-id=1999%3A45397#184882743

### Acceptance Criteria
- [ ] Add 3 dot icon to webview bottom nav bar
- [ ] On press of icon, show bottom sheet. 
- [ ] On confirm from bottom sheet, launch dapp in default device browser. The url opened in external browser should be exactly where the user was in the webview. The webview can remain open in Valora, for when the user returns to the app.
- [ ] Add analytics event on button press to launch external browser (Kathy to fill in)

**Comment by nityarao** [comment]
Updated these designs! @kathaypacific

**Comment by MuckT** [comment]
@Kathy Should I send the active dapp or the current url? Someone could navigate to Ubeswap > Menu > Charts (Celo Tracker)? The active dapp would be Ubeswap; however, the page they are intending to open external browser would be Celo Tracker.

**Comment by kathaypacific** [comment]
> @Kathy Should I send the active dapp or the current url? On iOS someone could navigate to Ubeswap > Menu > Charts (Celo Tracker)? The active dapp would be Ubeswap; however, the page they are intending to open external browser would be Celo Tracker.

Yeah good question...it depends if we speculate that the scenario of a dapp linking to another dapp on our dapp list will be more popular going forward. for now, only Ubeswap does this so my recommendation would be to stick to just the active dapp. this is consistent with other events (e.g. the close webview event) - what do you think?

maybe it's a question for Ellen or Gowtham, what kind of data is most correct for analysis - if we do want to change to tracking dapp url instead of dapp name, we could make an update for all the events at once
